modules/ROOT/pages/export-to-word-with-jwt-authentication-nodejs.adoc

```diff
@@ -0,0 +1,63 @@
+= {pluginname} with JWT authentication (Node.js) Guide
+:navtitle: JWT Authentication setup for Export to Word
+:description: Guide on how to setup JWT Authentication for exporting docx files with Export to Word
+:keywords: jwt, authentication, exportword, node.js
+:pluginname: Export to Word
+:plugincode: exportword
+
+
+include::partial$auth/document-converters/intro-and-prerequisites.adoc[]
+
+include::partial$auth/document-converters/initial-project-setup.adoc[]
+
+== Setup
+
+include::partial$auth/document-converters/jwt-setup-document-converters.adoc[leveloffset=+1]
+
+. In the root directory, copy and paste the server setup code into the `jwt.js` file (refer to the *Configuring the Node.js Server for JWT Token Generation* section).
```

Comment:
> refer to the *Configuring the Node.js Server for JWT Token Generation* section

I'm not sure if this is intended to be a link, or it could be renamed to match a section somewhere in the page for clarity.

Reply:
Updated to `Server Setup (jwt.js)` instead.